testing/pavucontrol/APKBUILD

```diff
@@ -0,0 +1,36 @@
+# Contributor: Fredrick Brennan <copypaste@kittens.ph>
+# Maintainer: Fredrick Brennan <copypaste@kittens.ph>
+pkgname=pavucontrol
+pkgver="20171007"
+pkgrel=1
+pkgdesc="A graphical interface for changing the volume of PulseAudio streams and other PulseAudio related tasks"
+url="https://freedesktop.org/software/pulseaudio/pavucontrol/"
+arch="all"
+license="GPL2+"
+depends="pulseaudio dbus gtkmm3 libsigc++ libcanberra"
```

Comment:
I dont think you need the pulseaudio, gtkmm3, libsigc++ or libcanberra. abuild should automatically detect the shared library dependencies.

Reply:
You do. They are all linked, and `pavucontrol` won't start without having them. Libcanberra is used for the sound it makes when you change volume to tell you the current volume. Libsigc++ is also linked and it won't start. It uses GTKmm3 for its toolkit. It needs every library I listed to start. Yes, this could be considered "bloat". If you don't like it, don't use this package; I plan to also package `pulseaudio-mixer-cli` if this gets approved, which has fewer deps (only ncurses, pulseaudio and python3 I think).

Reply:
It also links against libpulse.so because it uses the PulseAudio API.

Reply:
It’s not needed to explicitly specify dependencies that are automatically detected by abuild. You can see it in the abuild log:

https://travis-ci.org/alpinelinux/aports/builds/287546330#L1329-L1350:
```
>>> pavucontrol*: Tracing dependencies...
	pulseaudio
	dbus
	gtkmm3
	libsigc++
	libcanberra
	so:libatkmm-1.6.so.1
	so:libc.musl-x86_64.so.1
	so:libcanberra-gtk3.so.0
	so:libcanberra.so.0
	so:libgcc_s.so.1
	so:libgdkmm-3.0.so.1
	so:libglib-2.0.so.0
	so:libglibmm-2.4.so.1
	so:libgobject-2.0.so.0
	so:libgtk-3.so.0
	so:libgtkmm-3.0.so.1
	so:libintl.so.8
	so:libpulse-mainloop-glib.so.0
	so:libpulse.so.0
	so:libsigc-2.0.so.0
	so:libstdc++.so.6
```